clients/cobol-lsp-vscode-extension/test-report.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Probably this file is here by mistake.

Reply:
Yes, this file must not be included in repo

Reply:
Fixed!